sdk/python/feast/infra/offline_stores/offline_store.py

```diff
@@ -152,36 +140,49 @@ def to_arrow(
 
         return pyarrow.Table.from_pandas(features_df)
 
+    @abstractmethod
+    def _to_df_internal(self) -> pd.DataFrame:
```

Comment:
The docs should clarify why this exists and is different form `to_df`.

Reply:
good catch